### PR TITLE
Update CRNN readme

### PR DESCRIPTION
**Pull request by hqkate**
针对之前文档中模糊不清的部分，细化了一些介绍：
1. 添加了 dataset usage 部分介绍各数据集的用途及评估的操作步骤以便读者复现结果
2. 评估结果中添加了一列说明 字典字符

[419更新] 
1. 增加了benchmark数据：IIIT, SVT, IC03, IC13, IC15, SVTP, CUTE 分别的精度
2. 增加了 configs/参数配置文档说明.md
3. 增加了默认字典及自定义字典说明

[420更新]
1. 添加了性能数据

文档在有一些冗余的部分，根据个人理解有一些删减，请review，thanks!

Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

**Review by SamitHuang — APPROVED** [review]
LGTM

File: configs/rec/crnn/README.md
Comment (SamitHuang) [inline configs/rec/crnn/README.md:1]: 这里补充下 training set, validation set分别多少张images.

Comment (SamitHuang) [reply configs/rec/crnn/README.md:1]: Inference的数据也合进去。分Training performance 和 Inference performance两张表。

**Review by SamitHuang — COMMENTED** [review]
need to merge as one commit. 
参数配置文档，单独提一个PR。

File: configs/参数配置文档说明.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+| num_columns_to_net | output_columns中网络construct函数的输入数量 | 1 | \ |
+| **loader** | 数据加载设置 ||
+| shuffle | 每个epoch是否打乱数据顺序 | True | True/False |
+| batch_size | 单卡的批量大小 | 64 | \ |
+| drop_remainder | 当数据总数不能除以batch_size时是否丢弃最后一批数据 | True | \ |
+| max_rowsize | 指定在多进程之间复制数据时，共享内存分配的最大空间 | 12 | 默认值：64 |
+| num_workers | 指定 batch 操作的并发进程数/线程数 | 8 | 该值应大于或等于2 |
+
+
+### 评估流程 (eval)
```
[the remark this answers is not in the document]
Comment (hqkate) [reply configs/参数配置文档说明.md:161]: 已更新并另开了一个单独的PR #212

**Review by HaoyangLee — APPROVED** [review]

**Pull request merged**